static/modules/entity.js

```diff
@@ -114,18 +128,76 @@ export class Entity {
     this.vel = vel;
     /** @type {Vector} */
     this.acc = acc;
+    this.collisionType = "Circle";
+  }
+
+  /**
+   * Set the collisionShape of the entity. Give no arguments to reset the shape
+   * to be calculated each call of getCollisonShape
+   * @param {CollisionShape | undefined} collisionShape
+   * @returns {void}
```

Comment:
I made the same nitpick earlier, but I kind of like when documentation comments don't have symbol names in them because it keeps it higher level and is easier to refactor (less likely to go out of date accidentally). I'd just say "collision shape" instead of collisionShape. This applies to documentation below too.